lib/util/zip.ex

```diff
@@ -41,21 +51,25 @@ defmodule Antikythera.Zip do
     end)
   """
   defun zip(context_or_epool_id :: v[EPoolId.t | Context.t],
+            cwd_raw_path        :: v[String.t],
             zip_raw_path        :: v[FileName.t],
-            src_raw_path        :: v[String.t],
+            src_raw_path        :: v[NonTraversalPath.t],
```

Comment:
Prohibited non-traversal path for `src_path`, in order to prevent from creating a *fish print* of directories outside `tmpdir`.

Reply:
> Prohibited non-traversal path for src_path, in order to prevent from creating a fish print of directories outside tmpdir.

In this sense, absolute paths like `/tmp...` are also problematic...

Reply:
>> Prohibited non-traversal path for src_path, in order to prevent from creating a fish print of directories outside tmpdir.

> In this sense, absolute paths like /tmp... are also problematic...

- Fixed:
  - Invalidated absolute path for src e9a3716
  - Remove auth for src path ddf6018

Reply:
> Invalidated absolute path for src

Note that `/` cannot be a `tmpdir` (though rarely zipped, anyway).